### PR TITLE
[SPARK-27486][Core][Test]Enable History server storage information test in the HistoryServerSuite

### PR DESCRIPTION
**Pull request by shahidki31**
## What changes were proposed in this pull request?

We have disabled a test related to storage in the History server suite after SPARK-13845. But, after SPARK-22050, we can store the information about block updated events to eventLog, if we enable "spark.eventLog.logBlockUpdates.enabled=true". 

   So, we can enable the test, by adding an eventlog corresponding to the application, which has enabled the configuration, "spark.eventLog.logBlockUpdates.enabled=true"

## How was this patch tested?
Existing UTs

### CODE REVIEW

**Comment by shahidki31** [comment]
Jenkins, test this please

**Comment by shahidki31** [comment]
retest this please

**Comment by shahidki31** [comment]
cc @srowen @vanzin Kindly review.

Could you please tell me why is it failing in the RAT test? Does eventLogs require Apache license headers

```
Could not find Apache license headers in the following files:
 !????? /home/jenkins/workspace/SparkPullRequestBuilder@3/core/src/test/resources/spark-events/local-1555466577611
[error] running /home/jenkins/workspace/SparkPullRequestBuilder@3/dev/check-license ; received return code 1
```

File: core/src/test/resources/spark-events/local-1555466577611 [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{"Event":"SparkListenerLogStart","Spark Version":"3.0.0-SNAPSHOT"}
```
Comment (wangyum) [inline core/src/test/resources/spark-events/local-1555466577611]: Add local-1555466577611 to https://github.com/apache/spark/blob/master/dev/.rat-excludes

Comment (shahidki31) [reply core/src/test/resources/spark-events/local-1555466577611]: Thanks @wangyum , I'll update

**Comment by shahidki31** [comment]
Jenkins, retest this please

**Comment by shahidki31** [comment]
Jenkins, retest this please

**Comment by shahidki31** [comment]
cc @srowen @vanzin This PR is just enabling a test case which disabled long back. Kindly request to review

**Review by srowen — APPROVED** [review]
Seems reasonable

**Comment by shahidki31** [comment]
Thanks @srowen

**Review by felixcheung — APPROVED** [review]

**Comment by srowen** [comment]
Merged to master

**Comment by shahidki31** [comment]
Thanks @srowen @felixcheung

**Pull request closed without merging**